packages/thirdweb/src/wallets/in-app/web/lib/auth/oauth.ts

```diff
@@ -30,12 +30,18 @@ export const loginWithOauthRedirect = (options: {
   authOption: SocialAuthOption;
   client: ThirdwebClient;
   ecosystem?: Ecosystem;
+  redirectUrl?: string;
```

Comment:
What is this being used for?

Reply:
It's being passed into `getLoginUrl` by the function `handleSocialLogin`. It's not used in `loginWithOauthRedirect` directly (since `options` is passed with the spread operator) but omitting it would give the error `Object literal may only specify known properties, and 'redirectUrl' does not exist in type`.

Reply:
`redirectUrl?: string` is already an argument on `getLoginUrl` since redirect is supported already for native